src/main/java/appeng/tile/AEBaseInvTile.java

```diff
@@ -19,223 +19,122 @@
-	public int getSizeInventory()
-	{
-		return this.getInternalInventory().getSizeInventory();
-	}
-
-	@Override
-	public ItemStack getStackInSlot( final int i )
-	{
-		return this.getInternalInventory().getStackInSlot( i );
-	}
-
-	@Override
-	public ItemStack decrStackSize( final int i, final int j )
-	{
-		return this.getInternalInventory().decrStackSize( i, j );
-	}
-
-	@Override
-	public ItemStack removeStackFromSlot( final int i )
-	{
-		return ItemStack.EMPTY;
-	}
-
-	@Override
-	public void setInventorySlotContents( final int i, @Nullable final ItemStack itemstack )
-	{
-		this.getInternalInventory().setInventorySlotContents( i, itemstack );
-	}
-
-	/**
-	 * Returns if the inventory is named
-	 */
-	@Override
-	public boolean hasCustomInventoryName()
-	{
-		return super.hasCustomInventoryName();
-	}
-
-	@Override
-	public boolean hasCustomName()
-	{
-		return this.hasCustomInventoryName();
-	}
-
-	@Override
-	public int getInventoryStackLimit()
-	{
-		return 64;
-	}
-
-	@Override
-	public boolean isUsableByPlayer( final EntityPlayer p )
-	{
-		final double squaredMCReach = 64.0D;
-
-		return this.world.getTileEntity( this.pos ) == this && p.getDistanceSq( this.pos.getX() + 0.5D, this.pos.getY() + 0.5D,
-				this.pos.getZ() + 0.5D ) <= squaredMCReach;
 	}
 
 	@Override
-	public void openInventory( final EntityPlayer player )
+	public void getDrops( final World w, final BlockPos pos, final List<ItemStack> drops )
 	{
+		final IItemHandler inv = getInternalInventory();
 
-	}
-
-	;
-
-	@Override
-	public void closeInventory( final EntityPlayer player )
-	{
-
-	}
-
-	@Override
-	public boolean isItemValidForSlot( final int i, final ItemStack itemstack )
-	{
-		return true;
-	}
-
-	@Override
-	public abstract void onChangeInventory( IInventory inv, int slot, InvOperation mc, ItemStack removed, ItemStack added );
-
-	@Override
-	public int[] getSlotsForFace( final EnumFacing side )
-	{
-		final Block blk = this.world.getBlockState( this.pos ).getBlock();
-		if( blk instanceof AEBaseBlock )
+		for( int l = 0; l < inv.getSlots(); l++ )
 		{
-			return this.getAccessibleSlotsBySide( ( (AEBaseBlock) blk ).mapRotation( this, side ) );
+			final ItemStack is = inv.getStackInSlot( l );
+			if( !is.isEmpty() )
+			{
+				drops.add( is );
+			}
 		}
-		return this.getAccessibleSlotsBySide( side );
 	}
 
-	@Override
-	public boolean canInsertItem( final int slotIndex, final ItemStack insertingItem, final EnumFacing side )
-	{
-		return this.isItemValidForSlot( slotIndex, insertingItem );
-	}
-
-	@Override
-	public boolean canExtractItem( final int slotIndex, final ItemStack extractedItem, final EnumFacing side )
-	{
-		return true;
-	}
-
-	@Override
-	public void clear()
-	{
-		this.getInternalInventory().clear();
-	}
-
-	@Override
-	public int getField( final int id )
-	{
-		return 0;
-	}
-
-	@Override
-	public void setField( final int id, final int value )
-	{
-
-	}
+	/*
+	 * @Override
+	 * public boolean isUsableByPlayer( final EntityPlayer p )
+	 * {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)